docs/src/main/asciidoc/config-yaml.adoc

```diff
@@ -185,14 +182,14 @@ file has to be present in the root of the working directory relative to the Quar
 ├── my-app-runner
 ----
 
-The values from this file override any values from the regular `application.yaml` file if exists.
+The values from this file override any values from the regular `application.yaml` file if it exists.
 
 == Configuration key conflicts
 
-The MicroProfile Config specification defines configuration keys as an arbitrary `.`-delimited string. However,
-structured formats like YAML only support a subset of the possible configuration namespace. For example, consider the
-two configuration properties `quarkus.http.cors` and `quarkus.http.cors.methods`. One property is the prefix of another,
-so it may not be immediately evident how to specify both keys in your YAML configuration.
+The MicroProfile Config specification defines configuration keys as an arbitrary `.`-delimited string.
```

Comment:
```suggestion
The MicroProfile Config specification defines configuration keys as an arbitrary dot-separated string.
```

Reply:
For [delimiter-separated values](https://en.wikipedia.org/wiki/Delimiter-separated_values), lets preserve the "delimiter" term.